sandbox-server/app.js

```diff
@@ -11,12 +12,17 @@ app.get('/', (request, response) => {
 
 app.get('/search', (request, response) => {
   response.type('text/plain')
-  if(request.query.hasOwnProperty('q')) {
-    response.send(`You searched for: ${request.query.q}`)
+  if(request.query.hasOwnProperty('q') && request.query.q === 'doodads') {
+    response.send('You searched for: "doodads"')
```

Comment:
@deadlyicon Dang. My git-fu did the opposite of what I wanted. Don't know why I didn't catch this. Thanks.